index.html

```diff
@@ -11,13 +13,17 @@
     </aside>
     <div class='main'>
       <h2>The Main Content</h2>
-      <div class='left'>
+
+      <img src='https://jesse.sh/img/me.jpg'>
+
```

Comment:
the `img` tag is missing the `alt` attribute for text that would appear in case the image doesn't render.